modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DefaultDDMFormValuesFactory.java

```diff
@@ -103,6 +107,32 @@ protected Value createDefaultValue(DDMFormField ddmFormField) {
 				defaultValue.getString(_ddmForm.getDefaultLocale())));
 	}
 
+	private void _populate(
+		Consumer<DDMFormFieldValue> consumer, List<DDMFormField> ddmFormFields,
+		Map<String, List<DDMFormFieldValue>> ddmFormFieldValuesMap) {
+
+		if (ddmFormFields != null) {
+			ddmFormFields.forEach(
```

Comment:
@rodrigopaulino is it possible to put this `forEach` outside the `if` to buy some space? You can change the `if` condition to `ddmFormFields == null`, for example.

Reply:
Sure thing

Reply:
Hey, @carolmariaabb. I pushed your suggestion, but I struggle to see the benefit, because it didn't actually buy lines, just columns (meaning, the code pretty much is being read as before). Is there more to it that I'm failing to see? Thanks in advance 🙏🏻

Reply:
Hi @rodrigopaulino, I agree with you, I only suggested it because Brian likes to do it 😅 (as you can see in this [commit](https://github.com/liferay/liferay-portal/commit/d1cb0ab4f767f1bff63d6b52c093c6843cd6991c) and in this other [commit](https://github.com/liferay/liferay-portal/commit/9b2db33864e19443e0c127d5c17d73de4cf0acdf))

Reply:
For those I can kinda see the benefits on these, because they're actually buying lines.
On the first commit, [this](https://github.com/liferay/liferay-portal/commit/d1cb0ab4f767f1bff63d6b52c093c6843cd6991c#diff-f11ba522ebbed814fa28de08bb23fd155aa6c5713a7ca0fe4c43af99b1fcebd6L158-L159) becomes [this](https://github.com/liferay/liferay-portal/commit/d1cb0ab4f767f1bff63d6b52c093c6843cd6991c#diff-f11ba522ebbed814fa28de08bb23fd155aa6c5713a7ca0fe4c43af99b1fcebd6R161), and, on the second one, [this](https://github.com/liferay/liferay-portal/commit/9b2db33864e19443e0c127d5c17d73de4cf0acdf#diff-7284fa014265d76e9b273dea87d4c77a4ef679ed55c5d5dd352bb88febae0a77L41-L43) becomes [this](https://github.com/liferay/liferay-portal/commit/9b2db33864e19443e0c127d5c17d73de4cf0acdf#diff-7284fa014265d76e9b273dea87d4c77a4ef679ed55c5d5dd352bb88febae0a77R45-R46). So I guess readability improves...
It's not much, but at least they're getting some lines in return. I mean the ROI is not very good. 😆
But I completely agree with you: we should improve as desired where we can in order to avoid changes ahead.

Thanks for responding to my question, @carolmariaabb. I very much appreciate your insights. 🙏🏻